dts/riscv/ite/it8xxx2-wuc-map.dtsi

```diff
@@ -0,0 +1,377 @@
+			wucs = <&wuc9 BIT(6)>; /* GPB4 */
+		};
+		wuc_wu95: wu95 {
+			wucs = <&wuc9 BIT(7)>; /* GPC2 */
+		};
+
+		/* WUC group 10 */
+		wuc_wu96: wu96 {
+			wucs = <&wuc10 BIT(0)>; /* GPF0 */
+		};
+		wuc_wu97: wu97 {
+			wucs = <&wuc10 BIT(1)>; /* GPF1 */
+		};
+		wuc_wu98: wu98 {
+			wucs = <&wuc10 BIT(2)>; /* GPF2 */
+		};
+		wuc_wu99: wu99 {
+			wucs = <&wuc10 BIT(3)>; /* GPF3 */
+		};
+		wuc_wu100: wu100 {
+			wucs = <&wuc10 BIT(4)>; /* GPA7 */
+		};
+		wuc_wu101: wu101 {
+			wucs = <&wuc10 BIT(5)>; /* GPB0 */
+		};
+		wuc_wu102: wu102 {
+			wucs = <&wuc10 BIT(6)>; /* GPB1 */
+		};
+		wuc_wu103: wu103 {
+			wucs = <&wuc10 BIT(7)>; /* GPB3 */
+		};
+
+		/* WUC group 11 */
+		wuc_wu104: wu104 {
+			wucs = <&wuc11 BIT(0)>; /* GPB5 */
+		};
+		wuc_wu105: wu105 {
+			wucs = <&wuc11 BIT(1)>; /* GPB6 */
+		};
+		wuc_wu106: wu106 {
+			wucs = <&wuc11 BIT(2)>; /* GPB7 */
+		};
+		wuc_wu107: wu107 {
+			wucs = <&wuc11 BIT(3)>; /* GPC1 */
+		};
+		wuc_wu108: wu108 {
+			wucs = <&wuc11 BIT(4)>; /* GPC3 */
+		};
+		wuc_wu109: wu109 {
+			wucs = <&wuc11 BIT(5)>; /* GPC5 */
+		};
+		wuc_wu110: wu110 {
+			wucs = <&wuc11 BIT(6)>; /* GPD3 */
+		};
+		wuc_wu111: wu111 {
+			wucs = <&wuc11 BIT(7)>; /* GPD4 */
+		};
+
+		/* WUC group 12 */
+		wuc_wu112: wu112 {
+			wucs = <&wuc12 BIT(0)>; /* GPD5 */
+		};
+		wuc_wu113: wu113 {
+			wucs = <&wuc12 BIT(1)>; /* GPD6 */
+		};
+		wuc_wu114: wu114 {
+			wucs = <&wuc12 BIT(2)>; /* GPE4 */
+		};
+		wuc_wu115: wu115 {
+			wucs = <&wuc12 BIT(3)>; /* GPG0 */
+		};
+		wuc_wu116: wu116 {
+			wucs = <&wuc12 BIT(4)>; /* GPG1 */
+		};
+		wuc_wu117: wu117 {
+			wucs = <&wuc12 BIT(5)>; /* GPG2 */
+		};
+		wuc_wu118: wu118 {
+			wucs = <&wuc12 BIT(6)>; /* GPG6 */
+		};
+		wuc_wu119: wu119 {
+			wucs = <&wuc12 BIT(7)>; /* GPI0 */
+		};
+
+		/* WUC group 13 */
+		wuc_wu120: wu120 {
+			wucs = <&wuc13 BIT(0)>; /* GPI1 */
+		};
+		wuc_wu121: wu121 {
+			wucs = <&wuc13 BIT(1)>; /* GPI2 */
+		};
+		wuc_wu122: wu122 {
+			wucs = <&wuc13 BIT(2)>; /* GPI3 */
+		};
+		wuc_wu123: wu123 {
+			wucs = <&wuc13 BIT(3)>; /* GPG3 */
+		};
+		wuc_wu124: wu124 {
+			wucs = <&wuc13 BIT(4)>; /* GPG4 */
+		};
+		wuc_wu125: wu125 {
+			wucs = <&wuc13 BIT(5)>; /* GPG5 */
+		};
+		wuc_wu126: wu126 {
+			wucs = <&wuc13 BIT(6)>; /* GPG7 */
+		};
+
+		/* WUC group 14 */
+		wuc_wu128: wu128 {
+			wucs = <&wuc14 BIT(0)>; /* GPJ0 */
+		};
+		wuc_wu129: wu129 {
+			wucs = <&wuc14 BIT(1)>; /* GPJ1 */
+		};
+		wuc_wu130: wu130 {
+			wucs = <&wuc14 BIT(2)>; /* GPJ2 */
+		};
+		wuc_wu131: wu131 {
+			wucs = <&wuc14 BIT(3)>; /* GPJ3 */
+		};
+		wuc_wu132: wu132 {
+			wucs = <&wuc14 BIT(4)>; /* GPJ4 */
+		};
+		wuc_wu133: wu133 {
+			wucs = <&wuc14 BIT(5)>; /* GPJ5 */
+		};
+		wuc_wu134: wu134 {
+			wucs = <&wuc14 BIT(6)>; /* GPJ6 */
+		};
+		wuc_wu135: wu135 {
+			wucs = <&wuc14 BIT(7)>; /* GPJ7 */
+		};
+
+		/* WUC group 15 */
+		wuc_wu136: wu136 {
+			wucs = <&wuc15 BIT(0)>; /* GPL0 */
+		};
+		wuc_wu137: wu137 {
+			wucs = <&wuc15 BIT(1)>; /* GPL1 */
```

Comment:
I think pin names `GPL1`,  `GPL2`, and `GPL3` are causing the license check failure.  I don't know if the scancode check can be bypassed or waived.

An alternative would be to document these pins as `GPIO L0`,`GPIO L1`, etc.

Reply:
Done, thanks~